posture/serialnumber_stub.go

```diff
@@ -0,0 +1,10 @@
+// Copyright (c) Tailscale Inc & AUTHORS
+// SPDX-License-Identifier: BSD-3-Clause
+
+package posture
+
+// GetSerialNumber returns a list of any serial numbers
+// found on the client
```

Comment:
"any" reads a bit odd, and I think we should make it explicit that these are machine serial numbers, not some sort of other serial numbers we might find on the client.

```suggestion
// GetSerialNumber returns client machine serial number(s).
```